illumidesk/Chart.yaml

```diff
@@ -10,6 +10,14 @@ dependencies:
     import-values:
       - child: rbac
         parent: rbac
+  - name: postgresql
+    version: "9.8.4"
```

Comment:
If possible we need to upgrade this version to v12.x since we are using that version in production.

Reply:
Looks good, I wasn't aware that the charts don't support the latest versions. As commented in the Slack channel, we will just use this version for now and document the Postgres version we have tested the stack with.